### DISCUSSION TITLE
Pyscipt, Jupyter and matplotlib

### DISCUSSION THREAD

**Discussion by wigster** [custom-components/pyscript#355] [Q&A]
I have set up pyscript in Jupyter sucessfully. 

I would like visualise some calculations using matplotlib to make sure functions in my scripts are doing what I expect, but it doesn't appear to work. Is there some way of using it?

**Comment by solomonsanderson** [comment]
You will need to be more specific.  Did it throw any errors and what were they, any relevant code snippets and what you may have tried already. I would happily try and help if you can provide more information :).
